packages/ai/src/mcp.ts

```diff
@@ -0,0 +1,17 @@
+import { Function } from './function';
+import { Schema } from './schema';
+
+export type McpClientParams = {
```

Comment:
should these MCP specific types be removed from the core AI package now that we have the plugin changes needed to keep the MCP concepts in their own packages?

Reply:
disregard, somehow was looking at an older commit